src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -2426,31 +2435,65 @@ private void OnAutoEvoPredictionComplete(PendingAutoEvoPrediction run)
         autoEvoPredictionRunSuccessful = true;
 
         // Set the initial value
-        totalPopulationLabel.ResetInitialValue();
-        totalPopulationLabel.Value = run.PlayerSpeciesOriginal.Population;
+        totalEnergyLabel.ResetInitialValue();
+
+        results.GetPatchEnergyResults(run.PlayerSpeciesNew).TryGetValue(Editor.CurrentPatch,
+            out var newTotalPatchEnergyGathered);
 
-        totalPopulationLabel.Value = newPopulation;
+        var totalEnergyLabelText = totalEnergyLabel.GetNode<Label>(totalEnergyLabel.ValuePath);
+
+        if (newTotalPatchEnergyGathered != null)
+        {
+            totalEnergyLabel.Value = newTotalPatchEnergyGathered.TotalEnergyGathered;
+
+            var formatted = StringUtils.ThreeDigitFormat(newTotalPatchEnergyGathered.TotalEnergyGathered);
+            totalEnergyLabelText.Text =
+                $"{formatted} ({newPopulation})";
+        }
+        else
+        {
+            // TODO: Make indicator not show NaN when moved to a new patch
+            GD.PrintErr("Failed to get total patch energy gathered");
```

Comment:
Reading this logic, I now realize that there is a problem here: the total energy needs to be the total energy gathered, not any per-patch energy. I'll modify this to calculate the total energy gathered and show it properly.